dappnode/DNP_DAPPMANAGER
# Deprecate manifest-type releases (ON HOLD)

## Context

> Provide context of this PR: why it was created? what does it fixes? what new feature does it provides? ...
Old DAppNode packages were published using the ipfs hash of the manifest. This way was deprecated long time ago

## Approach

> Define the solution for the issue or feature that this PR aims to fix

This PR aims to simplify code, add some types and remove deprecated support that should be useless

**TODO**
- [ ] Run script against dnp.dappnode.eth and public.dappnode.eth and research for DAppNode packages distributed with the dappnode manifest hash. Consult dappnode team to merge this PR.
- [ ] Upload standard package for the tests `Resilience features`:
    - Prevent installing package from unsafe origin not signed
    - Remove a package without compose
    - Remove a package with a broken compose
    - Failing installation due to bad compose

## Test instructions


Check all the dappnode install lifecycle (install, update, remove) using either local and ipfs remote
